models/marts/attribution/order_attribution.sql

```diff
@@ -0,0 +1,132 @@
+{{
+    config(
+        materialized='table'
+    )
+}}
+
+with sessions as (
+
+    select
+
+        session_id,
+        ad_performance_id,
+        inferred_user_id,
+        session_start,
+        session_index,
+        device_type,
+        attribution_channel,
+        attribution_source,
+        attribution_campaign
+
+    from {{ref('snowplow_sessions_final')}}
+
+),
+
+orders as (
+
+    select * from {{ ref('orders_xf') }}
+    where customer_id is not null
+
+),
+
+sessions_joined as (
+
+    select
+
+        orders.*,
```

Comment:
I wouldn't include all fields from orders because it makes this attribution table hard to work with. the goal here is to have the order date, order id, and order amount. if more info is desired it can be joined to orders